wandb/sdk/internal/handler.py

```diff
@@ -689,7 +689,7 @@ def handle_request_run_start(self, record: Record) -> None:
             self._settings, interface=self._interface, run_proto=run_start.run
         )
 
-        if run_start.run.resumed:
+        if run_start.run.resumed or run_start.run.forked:
```

Comment:
Yeah I asked for a system test, but based on what @dannygoldstein said he can’t test it because it only works for v2. And the current test container needs some config value set to work

Reply:
meaningfully testing this is not feasible right now due to the state of v2 runs in our test env

Reply:
@timoffex maybe we can have a placeholder test with the `wandb_core_failure` marker? do you think it has value? this will allow us to refer to it

Reply:
I think it's better to have no test rather than a placeholder. Since it's hard to add a test, it's fine not to do so, but I hope we can make v2 run functionality testable soon